### DISCUSSION TITLE
Import HUD layout to default skin? Or fall back to Lazer skin elements?

### DISCUSSION THREAD

**Discussion by puyoxyz** [ppy/osu#15895] [Q&A] [answered]
I tried importing an OSK file with *just the JSON file with the HUD layout* and it works, but it's using the classic default skin. Is there a way to import a JSON HUD layout into the osu!lazer skin, or fall back to osu!lazer skin elements instead of the classic skin elements?

**Comment by peppy** [comment, marked as answer]
That sounds like a potential oversight in the export logic. Will take a look into this.

**Reply by puyoxyz** [reply]
(oh wait uh um it's um probably not that because i exported another skin that had a layout and deleted all the other files that werent the layout json so like it's probably like not something to do with exporting)

**Reply by peppy** [reply]
Tracking via #15896.
